src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -90,10 +93,13 @@ public Command parseCommand(String userInput) throws ParseException {
             return new ClearCommand();
 
         case FindCommand.COMMAND_WORD:
```

Comment:
Like jianyi said, i think we should consider doing this for all the other commands that apply to both suppliers and deliveries, like add, delete, find, list, etc